packages/addon-dev/src/rollup-public-entrypoints.ts

```diff
@@ -2,6 +2,10 @@ import walkSync from 'walk-sync';
 import type { Plugin } from 'rollup';
 import { join } from 'path';
 
+function normalizeFileExt(fileName: string) {
```

Comment:
hbs needs to be supported yeah?
for template-only components?

currently trying to figure out how to import template-only components.
stand by!

Reply:
so, here is what I'm dealing with atm:

Baseline commit: https://github.com/NullVoxPopuli/ember-addon-v2-typescript-demo/commit/c4bb6adab3a9045d697fe590938883dec956fef8

output of demo/index.js is this:
```js
//❯ cat dist/components/demo/index.js 
import { setComponentTemplate } from '@ember/component';
import { hbs } from 'ember-cli-htmlbars';
import { __decorate } from 'tslib';
import Component from '@glimmer/component';
import { tracked } from '@glimmer/tracking';

var TEMPLATE = setComponentTemplate(TEMPLATE, hbs`Hello there!

<out>{{this.active}}</out>

<this.Button @onClick={{this.flip}} />
`);

class Demo extends Component {
  constructor(...args) {
    super(...args);
    this.active = false;

    this.flip = () => this.active = !this.active;
  }

}

__decorate([tracked], Demo.prototype, "active", void 0);

setComponentTemplate(TEMPLATE, Demo);

export { Demo as default };

```

and then when I add nodeResolve in this commit: 

output is this (where did `@glimmer/application` come from?):
```js
//❯ cat dist/components/demo/index.js 
import { setComponentTemplate } from '@ember/component';
import { hbs } from 'ember-cli-htmlbars';
import { __decorate } from 'tslib';
import { t as tracked, C as Component } from '../../tracked-b2e133b9.js';
import '@glimmer/application';

var TEMPLATE = setComponentTemplate(TEMPLATE, hbs`Hello there!

<out>{{this.active}}</out>

<this.Button @onClick={{this.flip}} />
`);

var FlipButton = setComponentTemplate(FlipButton, hbs`<button {{on 'click' @onClick}}>
  flip
</button>
`);

class Demo extends Component {
  constructor(...args) {
    super(...args);
    this.Button = FlipButton;
    this.active = false;

    this.flip = () => this.active = !this.active;
  }

}

__decorate([tracked], Demo.prototype, "active", void 0);

setComponentTemplate(TEMPLATE, Demo);

export { Demo as default };
```

While playing around with this, I had to add `ember-cli-htmlbars` to the external list due to this error:
```
[build:*js] [!] Error: 'hbs' is not exported by ../node_modules/ember-cli-htmlbars/lib/index.js, imported by src/components/demo/index.hbs
```